google-login-plugin/src/com/google/gct/login/GoogleLogin.java

```diff
@@ -424,9 +425,9 @@ public boolean logOut(boolean showPrompt) {
     boolean loggedOut = activeUser.getGoogleLoginState().logOut(showPrompt);
     if(loggedOut) {
       logOutAllUsers();
+      UsageTrackerService.getInstance().trackEvent(LoginTracking.CATEGORY, LoginTracking.LOGIN, "logout.complete", null);
```

Comment:
could we rewrite logOut to use logOut(showPrompt)? Or remove logOut(showPrompt)? It doesn't appear to be used.